docs/src/pages/[platform]/theming/default-theme/icons/Colors.tsx

```diff
@@ -0,0 +1,26 @@
+import { Icon } from '@aws-amplify/ui-react';
+
+export function Colors() {
```

Comment:
Might be worth giving this a less generic name since this is a very broad namespace, for example `ColorsThemeIcon` (admittedly not a great name 😆 ). Same with on `Sizes` and `Typography`

Reply:
Yeah agreed, updated in latest.